docs/source/api/apollo-server.md

```diff
@@ -432,7 +432,7 @@ addMockFunctionsToSchema({
    Creates a client context(ClientInfo) based on the request pipeline's
    context, which contains values like the request, response, cache, and
    context. This generated client information will be provided to Apollo
```

Comment:
We seem to just be referring to 'Graph Manager' rather than 'Apollo Graph Manager' in most cases

```suggestion
   context. This generated client information will be provided to
```